src/ansys/dpf/core/field_definition.py

```diff
@@ -153,6 +153,61 @@ def dimensionality(self):
         self._api.csfield_definition_fill_dimensionality(self, dim, nature, dim.internal_size)
         return Dimensionality(dim.tolist(), natures(int(nature)))
 
+    def get_quantity_type(self, index=0):
```

Comment:
Maybe having just one property which returns the quantity_types would be easier too use, what do you think?